README.md

```diff
@@ -0,0 +1,4 @@
+# website
+Docs for choria found at choria.io
+amazing project
```

Comment:
lets make it `Source for documentation found at [choria.io](https://choria.io)`, you can push a 2nd commit to the same branch and that should update this PR